/**:
  ros__parameters:
    height: 720
    width: 1280
    distortion_model: 'rational_polynomial'
    d: [7.613864421844482, 17.2153263092041, 0.00012345814320724458, -0.00020271474204491824, -1.0174853801727295, 7.641190528869629, 19.72230339050293, 1.2936210632324219, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    k: [761.81488037, 0.0, 646.52478027, 0.0, 761.15325928, 361.41662598, 0.0, 0.0, 1.0]  # For 1280x720
    p: [761.81488037, 0.0, 646.52478027, 0.0, 0.0, 761.15325928, 361.41662598,0.0, 0.0, 0.0, 1.0, 0.0]  # For 1280x720
